odl/space/__init__.py

```diff
@@ -15,59 +15,9 @@
+"""Concrete vector spaces."""
 
-Spaces of n-tuples (modules `ntuples`, `cu_ntuples`)
-====================================================
-
-NumPy implementation (module `ntuples`)
----------------------------------------
-
-+----------------------+-----------------------------------------------+
-|Name                  |Description                                    |
-+======================+===============================================+
-|`Ntuples`             |Set of n-tuples of any NumPy supported type    |
-+----------------------+-----------------------------------------------+
-|`FnBase`              |**Abstract** base class for spaces of n-tuples |
-|                      |over the real or complex numbers               |
-+----------------------+-----------------------------------------------+
-|`Fn`                  |Space of n-tuples over the real or complex     |
-|                      |numbers allowing any adequate scalar data type |
-+----------------------+-----------------------------------------------+
-|`Cn`                  |Space of n-tuples of complex numbers           |
-+----------------------+-----------------------------------------------+
-|`Rn`                  |Space of n-tuples of real numbers              |
-+----------------------+-----------------------------------------------+
-
-CUDA implementation (module `cu_ntuples`)
------------------------------------------
-
-Requires the compiled extension `odlpp`
-
-+----------------------+-----------------------------------------------+
-|Name                  |Description                                    |
-+======================+===============================================+
-|`CudaNtuples`         |Set of n-tuples of any type supported by the   |
-|                      |`odlpp` backend                                |
-+----------------------+-----------------------------------------------+
-|`CudaFn`              |Space of n-tuples over the real or complex     |
-|                      |numbers allowing any adequate scalar data type |
-+----------------------+-----------------------------------------------+
-|(`CudaCn`)            |Space of n-tuples of complex numbers (TODO)    |
-+----------------------+-----------------------------------------------+
-|`CudaRn`              |Space of n-tuples of real numbers              |
-+----------------------+-----------------------------------------------+
-
-Function spaces (module `fspace`)
-=================================
-
-+----------------------+-----------------------------------------------+
-|Name                  |Description                                    |
-+======================+===============================================+
-|`L2`                  |Square-integrable functions taking real or     |
-|                      |complex values                                 |
-+----------------------+-----------------------------------------------+
-"""
+# TODO: write an introduction
```

Comment:
See above. Just wanted to check how we should proceed here. Some words rather than tables, right?